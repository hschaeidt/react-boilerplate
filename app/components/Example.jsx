import React from 'react';

export default () => (
  <p>Example.jsx</p>
);
