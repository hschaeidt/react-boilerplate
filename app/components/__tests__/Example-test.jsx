import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Example from '../Example';

describe('<Example />', () => {
  it('render correctly', () => {
    const tree = renderer.create(<Example />);
    expect(tree).toMatchSnapshot();
  });

  describe('render', () => {
    it('should render a paragraph', () => {
      const clock = shallow(<Example />);
      const actualText = clock.find('p').text();

      expect(actualText).toBe('Example.jsx');
    });
  });
});
