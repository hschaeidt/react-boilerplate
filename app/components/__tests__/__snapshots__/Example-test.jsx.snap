exports[`<Example /> render correctly 1`] = `
<p>
  Example.jsx
</p>
`;
